cc/file/path.cc

```diff
@@ -14,10 +14,20 @@
 
 #include "cc/file/path.h"
 
+#include "absl/strings/match.h"
 #include "absl/strings/str_cat.h"
 
 namespace minigo {
 namespace file {
+
+#ifdef _WIN32
```

Comment:
no absl lib for this i guess

Reply:
Not yet. It's wrapped up with the rest of the absl::File API, which is on the road map to release but not for quite some time.